### DISCUSSION TITLE
How to install Silicon ready apps only?

### DISCUSSION THREAD

**Discussion by tulik** [Homebrew/discussions#560] [Everyday usage]
Is there a flag or another way I can determine if the app runs M1 natively? I would like to resign from installing apps which don't support Apple Silicon.

**Comment by carlocab** [comment]
If you've installed `brew` into `/opt/homebrew`, then `brew install` (without any additional flags) will only install formulae that run M1 natively.

**Reply by SMillerDev** [reply]
Please note this doesn't work for casks like Google Chrome etc
